Assets/Scripts/Enemy.cs

```diff
@@ -21,20 +21,20 @@ public class Enemy : Entity
     private float fireRateTimer = 0;
     private IWeapon fireableWeapon;
     private bool movingEnabled;
+    public bool movingEnemy = true;
     private CharacterController cc;
 
     private void Awake()
     {
+        MaxHealth = this.maxHealth;
```

Comment:
Why move these lines here? Just curious

Reply:
Oh, I'm not sure to be honest. I was having a lot of trouble getting spawn points to override their enemies' stats, so I was playing around with different parts of the code trying to see what was causing the issue. Hope this didn't complicate anything for you.